docs/sources/access/index.md

```diff
@@ -0,0 +1,67 @@
+---
+description: Access or install Explore Profiles.
+canonical: https://grafana.com/docs/grafana/latest/explore/simplified-exploration/profiles/access/
+keywords:
+  - Install
+  - Configure
+  - Explore Profiles
+title: Access or install Explore Profiles
+menuTitle: Access or install
+weight: 150
+refs:
+  pyroscope-data-source:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/datasources/pyroscope/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/connect-externally-hosted/data-sources/pyroscope/
+---
+
+# Access or install Explore Profiles
+
+Explore Profiles is a native Grafana application designed to integrate seamlessly with [Pyroscope](https://github.com/grafana/pyroscope), the open source continuous profiling platform, providing a smooth, query-less experience for browsing and analyzing profiling data.
```

Comment:
Not too sure i like query-less, we still do queries, the user just doesn't use a query language.

```suggestion
Explore Profiles is a native Grafana application designed to integrate seamlessly with [Pyroscope](https://github.com/grafana/pyroscope), the open source continuous profiling platform, providing a smooth, query-language-less experience for browsing and analyzing profiling data.
```

Reply:
I believe there is ongoing discussion about the use of the word "queryless" but the spelling we've adopted until that's resolved is without a hyphen: https://github.com/grafana/writers-toolkit/blob/main/vale/dictionary.jsonnet#L167

Reply:
Christian, Your suggestion is more technically accurate. We're having the same type of conversation with the other apps as well. The Traces app uses queries underneath. Users can't see the TraceQL behind it.  

```suggestion
Explore Profiles is a native Grafana application designed to integrate seamlessly with [Pyroscope](https://github.com/grafana/pyroscope), the open source continuous profiling platform, providing a smooth, queryless experience for browsing and analyzing profiling data.
```